app/static/styles/main.css

```diff
@@ -144,4 +151,35 @@ body.resume header div div a.resume, body.contact header div div a.contact {
     color: white;
     font-size: 50px;
     font-weight: 700;
+}
+
+
+/* for About Page */
+
+.about {
+    background-color: #56bfc9;
+}
+
+.about img {
+    width: 100%;
+    height: 100%;
+    background-color: #d9d9d9;
+    border-radius: 50px;
+}
+
+.about table {
+    padding-left: 5%;
+    padding-top: 2rem;
+    padding-bottom: 1rem;
+}
+
+.about h1 {
+    font-size: 3vw;
```

Comment:
I feel like it might be better to make the font size constant (like in px or something) instead of variable on the screen size? It gets super small on a smaller screen so it might be hard to read.

Reply:
done, switched to px.